themes/default/content/docs/support/troubleshooting.md

```diff
@@ -196,6 +196,24 @@ You may encounter an error when you downgrade provider versions _after_ your sta
 If you must downgrade the version of a provider your `pulumi` program depends on, you will need to [manually edit your deployment](#editing-your-deployment)
 and change the version of the provider your stack depends on and then import that as the latest state of your stack.
 
+### Nothing happens when running Pulumi
+
+You run Pulumi and nothing happens, with output resembling like this:
+
+```
+$ pulumi up
+Previewing update (<stack name>):
+
+Resources:
+
+$
+```
+
+You probably have a system-wide proxy server running on your machine. The [Pulumi architecture ](https://www.pulumi.com/docs/intro/concepts/how-pulumi-works/)
+has 3 different components, running as separate processes which talk to each other using a bidirectional gRPC protocol
+on IP address `127.0.0.1`. Your proxy server should be configured **NOT** to proxy
+these local network connections. Check if you can add both `127.0.0.1` and `localhost` to the exclusion list of your proxy server.
```

Comment:
```suggestion
If you have a system-wide proxy server running on your machine, it may be misconfigured. The [Pulumi architecture ](https://www.pulumi.com/docs/intro/concepts/how-pulumi-works/)
has three different components, running as separate processes which talk to each other using a bidirectional gRPC protocol
on IP address `127.0.0.1`. Your proxy server should be configured **NOT** to proxy
these local network connections. Add both `127.0.0.1` and `localhost` to the exclusion list of your proxy server.
```